conda/common/configuration.py

```diff
@@ -709,14 +728,6 @@ def __init__(self, name, value, element_type, key_flag, value_flags, validation=
         super(SequenceLoadedParameter, self).__init__(
             name, value, key_flag, value_flags, validation)
 
-    def __eq__(self, other):
-        if type(other) is type(self):
-            return self.value == other.value
-        return False
-
-    def __hash__(self):
-        return hash(self.value)
-
```

Comment:
Are these needed for comparison elsewhere?

Reply:
remove it bc it is the same as the superclass